src/options/proof_options.toml

```diff
@@ -139,3 +139,11 @@ name   = "Proof"
   type       = "bool"
   default    = "false"
   help       = "Check proof steps for satisfiability, for refutation soundness testing. Note this checks only steps for non-core proof rules"
+
+[[option]]
+  name       = "lfscExpandTrust"
+  category   = "expert"
+  long       = "lfsc-expand-trust"
+  type       = "bool"
+  default    = "false"
+  help       = "Print the children of trusted proof steps"
```

Comment:
Do we need this option? Can't we always print the children of trusted proof steps?

Reply:
There is a tradeoff, they are unnecessary for proof checking, but they ensure that the entire proof is reflected in the output.